src/dev/code_coverage/ingest_coverage/index_mapping.md

```diff
@@ -0,0 +1,91 @@
+# Create index mapping 
+
+This is usually done in Kibana's dev tools ui.
+
+```
+PUT kibana_coverage
+{
+  "mappings": {
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        },
+        "branches": {
+          "properties": {
+            "covered": {
+              "type": "long"
+            },
+            "pct": {
+              "type": "long"
+            },
+            "skipped": {
+              "type": "long"
+            },
+            "total": {
+              "type": "long"
+            }
+          }
+        },
+        "functions": {
+          "properties": {
+            "covered": {
+              "type": "long"
+            },
+            "pct": {
+              "type": "long"
+            },
+            "skipped": {
+              "type": "long"
+            },
+            "total": {
+              "type": "long"
+            }
+          }
+        },
+        "lines": {
+          "properties": {
+            "covered": {
+              "type": "long"
+            },
+            "pct": {
+              "type": "long"
+            },
+            "skipped": {
+              "type": "long"
+            },
+            "total": {
+              "type": "long"
+            }
+          }
+        },
+        "path": {
```

Comment:
Is this out of date with the code?  In the data I see `coveredFilePath` but not in this mapping

Reply:
yeah, I'll fixup.  Thanks boss.